cvat/apps/engine/static/engine/js/annotationParser.js

```diff
@@ -25,9 +24,7 @@ class AnnotationParser {
     }
 
     _getBoxPosition(box, frame) {
-        frame = Math.min(frame - this._startFrame, this._im_meta.length - 1);
-        const imWidth = this._im_meta[frame].width;
-        const imHeight = this._im_meta[frame].height;
+        const { width: imWidth, height: imHeight } = window.cvat.job.getImageSize(frame);
```

Comment:
just curious: why size computation changed?
in particular, `_im_meta` looks like a good place to store URL and other additional parameters

Reply:
I've changed the code in order to use existing size computation approach for tasks stored in cvat and different approach for tasks stored in records service. I didn't want to use existing code because it looks really fishy - the back-end sends `_im_meta` as an array of meta-data for all the images in the task, but the front-end treats it as if it's an array of meta-data of one particular job, so wrong image size gets picked up if the images have different sizes (it seems the authors haven't really tested this scenario).
The new approach is quite similar to the old one, but it doesn't have these flaws.